StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1023DereferenceAndAccessOfSymbolsMustBeSpacedCorrectly.cs

```diff
@@ -125,40 +137,27 @@ private static void HandleAsteriskToken(SyntaxTreeAnalysisContext context, Synta
                 allowAtLineStart = false;
                 allowAtLineEnd = true;
                 allowPrecedingSpace = false;
-                var nextToken = token.GetNextToken();
```

Comment:
A local variable or function named 'nextToken' is already defined in this scope

Reply:
This can be resolved with less code churn by removing `var`.